product-backend/src/main/java/org/kainos/ea/resources/AdminController.java

```diff
@@ -0,0 +1,40 @@
+package org.kainos.ea.resources;
+
+import io.swagger.annotations.Api;
+import org.checkerframework.checker.units.qual.A;
+import org.kainos.ea.api.AdminService;
+import org.kainos.ea.cli.Admin;
+import org.kainos.ea.client.FailedToCreateBandException;
+import org.kainos.ea.db.AdminDao;
+import org.kainos.ea.exception.NameTooShortException;
+import org.kainos.ea.validator.AdminValidator;
+import org.kainos.ea.db.DatabaseConnector;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+
+@Api("Admin API")
+@Path("/api")
+public class AdminController {
```

Comment:
Rename AdminController to BandController, apply it to DAO and frontend classes
```suggestion
public class BandController {
```